source/common/common/trie.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <memory>
```

Comment:
Ah, thanks for the pointer. I'll update the contributing doc. Most Googlers have been indoctrinated the opposite way.